scrapy/commands/parse.py

```diff
@@ -8,7 +8,8 @@
 from scrapy.item import BaseItem
 from scrapy.utils import display
 from scrapy.utils.conf import arglist_to_dict
-from scrapy.utils.spider import iterate_spider_output, spidercls_for_request
+from scrapy.utils.spider import iterate_spider_output
+from scrapy.utils.spider import spidercls_for_request
```

Comment:
Could you please keep these imports as-is? They are perfectly fine according to pep8 ("It's okay to say this though: from subprocess import Popen, PIPE").